apps/vanilla/src/pages/blackjack/ts/core/game.ts

```diff
@@ -0,0 +1,77 @@
+import { ResultDisplayer } from '../views/resultDisplayer';
+
+import { WinnerDisplayer } from '../views/winnerDisplayer';
+
+import { DebugDisplayer } from '../views/debugDisplayer';
+
+import { TurnGenerator } from './turnGenerator';
+import { Player } from './player';
+import { DiceGenerator } from './diceGenerator';
+import { Publisher } from './publisher';
+
+/**
+ * Game class that manages the game logic and UI updates.
+ */
+export class Game {
+	private readonly players: Player[];
```

Comment:
https://wiki.saritasa.rocks/frontend/languages/ts-js/best-practices#prefer-tuple-properties
```suggestion
	private readonly players: radonly Player[];
```